test/distributed/optim/test_named_optimizer.py

```diff
@@ -424,3 +427,8 @@ def test_init_state(self):
         self.assertFalse(
             torch.all(named_sd["state"]["net3.weight"]["momentum_buffer"]).item()
         )
+
+
+device = ["cpu", "cuda", "hpu", "xpu"]
```

Comment:
Why those devices and not say MTAI or PrivateUse1?

Also, this list seems repeated, in that case it needs to go to common file of sorts